Formula/tinyscheme.rb

```diff
@@ -20,35 +22,38 @@ def install
     share.install("init.scm")
     bin.install("scheme")
   end
+
+  test do
+    assert_match "Usage: tinyscheme", shell_output("#{bin}/scheme -?", 1)
```

Comment:
The CI seemed to have problems with the `hello.scm` I had originally (non repro on Mavericks), so @ilovezfs advised changing it.